rest_api/static/rest_api/openapi/openapi-schema.yml

```diff
@@ -2404,32 +2432,34 @@ paths:
                     type: array
                     description: "List of all PGS Catalog publications"
                     items:
-                      type: object
-                      properties:
-                        title:
-                          type: string
-                          description: "Title of the publication"
-                          example: 'The Polygenic Score Catalog as an open database for reproducibility and systematic evaluation'
-                        doi:
-                          type: string
-                          description: "doi of the publication"
-                          example: '10.1038/s41588-021-00783-5'
-                        PMID:
-                          type: number
-                          description: "PubMed ID"
-                          example: 33692568
-                        authors:
-                          type: string
-                          description: "Publication first author(s)"
-                          example: 'Lambert et al'
-                        journal:
-                          type: string
-                          description: "Name of the journal"
-                          example: 'Nature Genetics'
-                        year:
-                          type: number
-                          description: "Year of publication"
-                          example: 2021
+                      $ref: '#/components/schemas/PGSCatalogPublication'
```

Comment:
Do we need more than 1 example for that ? I though we would only display the latest publication for the REST API

Reply:
Initially I found weird to have 2 times the same publication, but I can roll it back to previous format

Reply:
Yes, I don't think we need all the examples there. Just one in an array should be enough